### ISSUE TITLE
Is this Asset still being maintained?

### ISSUE THREAD

**Issue by derwaldgeist** [universal-tools/UTNotificationsFeedback#140]
There's a lot of issues reported here, and radio-silence since two years, on both the forums and in the issue tracker. The Asset is still on sale on the Asset store, though. So I am wondering what is going on? We *need* an update for Unity 2020+, since 2019 ran out of support.

**Comment by nathando** [comment]
It's really unprofessional that they just abandoned the whole project

**Comment by derwaldgeist** [comment]
I agree. They are even still selling it on the Unity Asset Store. Our app upgrade to Unity 2020+ is currently blocked because of this package. Is there any other solution out there? I would also be fine if they just open-sourced it, so we could at least try to repair the package.
